test/test_alert_types.cpp

```diff
@@ -275,6 +276,17 @@ TORRENT_TEST(session_stats_alert)
 	TEST_CHECK(v->message().find("session stats (") != std::string::npos);
 }
 
+TORRENT_TEST(alert_name)
+{
+	TEST_EQUAL(alert_name(tracker_announce_alert::alert_type), "tracker_announce"_sv);
```

Comment:
I think this test can be included in the macro `TEST_ALERT_TYPE` of `TORRENT_TEST(alerts_types)` to cover all tests